### DISCUSSION TITLE
hell release

### DISCUSSION THREAD

**Discussion by Oliver-Weng-Sorcerer** [Automattic/mongoose#11073] [General]
what's up with this hell release ?
https://github.com/Automattic/mongoose/releases

![Screen Shot 2021-12-10 at 3 09 59 PM](https://user-images.githubusercontent.com/28814939/145635336-76fd9435-a7a0-435e-93ef-4f1a7decf815.png)

any insights?

@IslandRhythms

**Comment by dandv** [comment]
Same question. @vkarpov15 ?

**Comment by IslandRhythms** [comment]
Sorry about that, Currently working on automating releases

**Comment by vkarpov15** [comment]
Accidental release while we were working on automating release generation. We deleted the release. Sorry for the trouble!
